.circleci/config.yml

```diff
@@ -0,0 +1,41 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:10.14.2-browsers
+
+      # Specify service dependencies here if necessary
+      # CircleCI maintains a library of pre-built images
+      # documented at https://circleci.com/docs/2.0/circleci-images/
+      # - image: circleci/mongo:3.4.4
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
+            # fallback to using the latest cache if no exact match is found
+            # - v1-dependencies-
+
+      - run: yarn
+
+      - save_cache:
+          paths:
+            - node_modules
+            - ~/.cache/Cypress
+          key: v1-dependencies-{{ checksum "package.json" }}
+
+      # run tests!
+      - run: yarn test
+
+      # Upload reports to Codecov
+      - run: bash <(curl -s https://codecov.io/bash)
```

Comment:
at this is line make sure you have set an environment variable for `CODECOV_TOKEN` in `CI` services like `CircleCI` or `Travis`

Reply:
We haven't done it yet, have we?
So far I didn't see that Codecov is integrated into our build yet.

Reply:
I'm doing it in `feature/codecov` branch

Reply:
Resolved in #14